### DISCUSSION TITLE
Customize a specific code block background color ?

### DISCUSSION THREAD

**Discussion by smarie** [squidfunk/mkdocs-material#3257] [Q&A] [answered]
Hi there, first of all thanks so much for this project ! I use it is all my open source python libs.

I am currently working on porting the sphinx-gallery project to mkdocs. 

I wish to customize the background color of code blocks to distinguish the console outputs (background color #fafae2) from the code inputs (standard color from theme). I am stuck trying to set attributes on the markdown code block: I tried adding my custom class `{: .mkd-glr-script-out-disp }` in several positions before, after, and in the code block header but it does not seem to be correctly taken into account.

Is this possible ? Thanks a lot in advance !

EDIT: To illustrate what I would like to reproduce, you may see the "out" part of this page: https://scikit-learn.org/stable/auto_examples/linear_model/plot_ols.html#sphx-glr-auto-examples-linear-model-plot-ols-py

**Comment by squidfunk** [comment, marked as answer]
The syntax for code blocks is:

```` markdown
```{ .python .extra-class }
import hello_world
```
````

See the Python Markdown Extensions docs for more guidance:
https://facelessuser.github.io/pymdown-extensions/extensions/superfences/#injecting-classes-ids-and-attributes

**Reply by smarie** [reply]
Thanks for the fast answer @squidfunk ! I really appreciate.
